### PR TITLE
Disable fast livesync with hook

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
Add before livesync hook to be able to control the livesync data. Added property in livesync service to force full sync which will be passed to the livesync data.

### CODE REVIEW

File: lib/services/livesync/livesync-service.ts [hunk 1]
```diff
@@ -50,15 +53,22 @@ class LiveSyncService implements ILiveSyncService {
 
 			this._isInitialized = true; // If we want before-prepare hooks to work properly, this should be set after preparePlatform function
 
-			let platformData = this.$platformsData.getPlatformData(platformLowerCase);
-			this.ensureAndroidFrameworkVersion(platformData).wait();
+			this.liveSyncCore(platform).wait();
+		}).future<void>()();
+	}
 
+	@helpers.hook('before-livesync')
```
Comment (rosen-vladimirov) [inline lib/services/livesync/livesync-service.ts]: this should be `livesync` not `before-livesync`

**Comment by rosen-vladimirov** [comment]
:+1:  after fixing livesync hook name and merging branch in common lib

**Comment by rosen-vladimirov** [comment]
Please retarget to release branch

**Comment by TsvetanMilanov** [comment]
Closing and retarget to release.

**Pull request closed without merging**